src/api/generic_api.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
+from config import Config
+import json
+import requests
+
+class GenericApi:
```

Comment:
The "correct" way would be to create an api interface class (class IApi) which implements all the methods that should be implemented by as we have it e.g. with i_char.py. 
I mean, I do not expect that we have many more API interface implementations. Still I would like to have it done properly.